### ISSUE TITLE
Support RTL languages

### ISSUE THREAD

**Issue by Manishearth** [rust-lang/www.rust-lang.org#869]
We have Fluent support for localizing the website, but I suspect the website would look terrible for any RTL language.

CSS is powerful enough that we should be able to do this _just_ using CSS (i.e, we shouldn't have to generate different per-language layouts, we generate the same layout with different CSS being applied).

One way to start is to generate a testing localization like `xx-AU` that's basically English marked as RTL. (Android used to have a cibarA localization for this). Then, start going through and seeing what's broken, and fixing it with the appropriate CSS.

It might just help to start an Arabic/Hebrew translation team _first_ so they can start reporting bugs.

**Comment by YasirAsiri** [comment]
I can help with Arabic

**Comment by Manishearth** [comment]
@YasirAsiri For actually translating things you'd have to form a team (https://github.com/rust-lang/www.rust-lang.org/blob/master/TRANSLATIONS.md).

This issue is more about the technical changes to the design necessary to make RTL languages work. But having a significant portion of the website already translated would be nice.

**Comment by Manishearth** [comment]
Also, make sure you're on the same page about which kind of Arabic. Preferably Modern Standard Arabic if possible.

**Comment by YasirAsiri** [comment]
@Manishearth Alright, I will see if I can form the team soon

**Comment by skade** [comment]
@yasirAsiri It is still perfectly fine to use arabic for experimentation/validation, to be clear.

**Comment by Manishearth** [comment]
Yeah, what I can do is set up an `xx-LTR` language on Pontoon for testing, and you can start translating things yourself. Alternatively if you can find three people you can set up a real team for Arabic.

**Comment by Goudarz** [comment]
Hi, I'm currently working on fixing right-to-left (RTL) rendering issues for the Persian (`fa`) language.

To make RTL support more robust and consistent across all RTL languages, I’d like to suggest adding `dir="rtl"` to the `<html>` tag when the selected language is a right-to-left one (like `fa`, `ar`, etc.). This small change would allow all RTL languages to benefit from correct base directionality without requiring manual fixes.

I’ve also contacted the [localization team](mailto:localization@rust-lang.org) via email, but I haven’t received a reply yet.

**Comment by senekor** [comment]
@Goudarz We are currently not accepting new languages.

**Comment by Goudarz** [comment]
@skade I understand that new languages are not being accepted right now, but since parts of the website have already been translated into Persian, our team would now want to continue the existing localization.
